README.md

```diff
@@ -0,0 +1,18 @@
+# Contact Tracer Application
+<i> <ins> Last modified: October 14, 2020 </ins> </i>
```

Comment:
You might not want to include this, git keeps track of this. Additionally, it's just another thing that needs to be updated manually.

Reply:
If you want to fill this space below your header usually badges work great! (Google github badges) Once you know versions of current framework and languages you'll be using, this is a very good place to mention them.

Reply:
Awesome thanks for the feedback-- I'll remove the "Last Modified" and add some badges for what we have so far.